### PR TITLE
Move README.md to README.html conversion for Apple into build process

### PR DESCRIPTION
**Pull request by reikdas**
Following up on https://github.com/root-project/root/pull/6832#pullrequestreview-533638738

The downside for this is that `textutil` will be a requirement for macOS, even if ROOT is not being packaged on that computer.

### CODE REVIEW

File: cmake/modules/RootCPack.cmake [hunk 1]
```diff
@@ -29,12 +29,23 @@ string(REGEX REPLACE "^([0-9]+).*$" "\\1" CXX_MAJOR ${CMAKE_CXX_COMPILER_VERSION
 string(REGEX REPLACE "^([0-9]+)\\.([0-9]+).*$" "\\2" CXX_MINOR ${CMAKE_CXX_COMPILER_VERSION})
 
 #---Resource Files-----------------------------------------------------------------------------------
-configure_file(README.md README.md COPYONLY)
 configure_file(LICENSE LICENSE.txt COPYONLY)
 configure_file(LGPL2_1.txt LGPL2_1.txt COPYONLY)
-set(CPACK_PACKAGE_DESCRIPTION_FILE "${CMAKE_BINARY_DIR}/README.md")
-set(CPACK_RESOURCE_FILE_LICENSE "${CMAKE_BINARY_DIR}/LICENSE.txt")
```
Comment (Axel-Naumann) [inline cmake/modules/RootCPack.cmake:36]: Why did you remove this?

**Review by Axel-Naumann — CHANGES_REQUESTED** [review]
Nice, much simpler! I see that textutil is part of macOS since 10.4 so we should be good.

**Comment by Axel-Naumann** [comment]
Did you check the output of `textutil`? I don't think it actually converts...?

**Comment by reikdas** [comment]
> Did you check the output of `textutil`? I don't think it actually converts...?

Works for me.

```
Mac-1605777847106:root runner$ ls | grep README
README
README.md
Mac-1605777847106:root runner$ textutil -convert html README.md -output README.html
Mac-1605777847106:root runner$ ls | grep README
README
README.html
README.md
```

File: cmake/modules/RootCPack.cmake (hunk 1 above)
Comment (reikdas) [reply cmake/modules/RootCPack.cmake:36]: Oops. My mistake. Should be fixed now.

**Comment by Axel-Naumann** [comment]
But please have a look at the rendered README.html. This is basically just adding `<html></html>` around the file... See my updated comment above; we need to do proper markdown-to-html conversion.

**Review by Axel-Naumann — APPROVED** [review]
